mongodb/doc/MongoDataset.md

```diff
@@ -0,0 +1,9 @@
+# MongoDB Dataset
+
+The MongoDB Dataset is a read only [dataset](/doc/builtin/datasets/Datasets.md)
+based on a MongoDB collection. It is meant to be used as a bridge between MLDB
+and MongoDB by allowing MLDB SQL queries to run over a MongoDB collection.
+
+## Configuration
+
+![](%%config dataset mongodb.dataset)
```

Comment:
Should we document the fact that the implementation is currently very poor and that in order to have better performance users should import mongodb collections into MLDB?